src/index.js

```diff
@@ -17,7 +17,6 @@ import { RoutingProvider } from './routing'
 import { ConsoleVisibleProvider } from './apps/Console/useConsole'
 import initializeSentryIfEnabled from './sentry'
 import { HelpScoutProvider } from './components/HelpScoutBeacon/useHelpScout'
-import { ClientBlockNumberProvider } from './components/AccountModule/useClientBlockNumber'
```

Comment:
As far as I could tell, there were no uses of this provider's value, as we only show the wallet's sync block (and each connectivity check directly calculates the block drift between the wallet and the read provider).